### PR TITLE
Fix conduit build with mirage3

### PR DESCRIPTION
**Pull request by avsm**

### CODE REVIEW

**Comment by rgrinberg** [comment]
@avsm could you rebase on master?

**Comment by avsm** [comment]
merged against master

File: opam [hunk 1]
```diff
@@ -7,7 +7,7 @@ bug-reports:  "https://github.com/mirage/ocaml-conduit/issues"
 tags:         "org:mirage"
 license:      "ISC"
 
-build:   [["./configure"] [make]]
+build:   [["env" "CONDUIT_DISCOVER_VERBOSE=1" "./configure"] [make]]
```
Comment (rgrinberg) [inline opam]: I don't mind if we get rid of this variable and always write the "verbose" output as well.

**Comment by samoht** [comment]
Travis errors are due to missing depexts on `m4` on Fedora and Alpine.

**Comment by samoht** [comment]
#196 should be merged into master, and I'll rebase this PR on `mirage-dev.

**Comment by samoht** [comment]
Rebased on `mirage-dev` in #197

**Pull request closed without merging**